engine/src/main/java/org/pentaho/di/trans/steps/fileinput/text/TextFileInputMeta.java

```diff
@@ -1384,9 +1384,9 @@ public void resolve( Bowl bowl ) {
       if ( inputFiles.fileName[i] != null && !inputFiles.fileName[i].isEmpty() ) {
         try {
           FileObject fileObject = KettleVFS.getFileObject( getParentStepMeta().getParentTransMeta().environmentSubstitute( inputFiles.fileName[i] ) );
-          if ( AliasedFileObject.isAliasedFile( fileObject ) ) {
```

Comment:
We've determined it's dead code and code that hasn't existed in the baseline for several releases. So I don't see any benefit to adding in commented dead code.

Reply:
Additionally, I don't see the value in keeping any of the changes for `TextFileInputMeta#resolve` nor adding the `ResolvableResource` class to implement.s